test/parameter_tests.jl

```diff
@@ -1,8 +1,6 @@
 using Test
 import CLIMAParameters as CP
-
-include("../src/utils/cli_options.jl")
-include("../parameters/create_parameters.jl")
+import ClimaAtmos as CA
 
 FT = Float64
 
```

Comment:
I can't seem to comment on collapsed lines in the PR but the two lines below need the `CA.` qualifier
```suggestion
s = CA.argparse_settings()
default_args = CA.cli_defaults(s)
```
Could be combined into one line as well: `default_args = CA.cli_defaults(CA.argparse_settings())`

Reply:
Otherwise, this looks good to me! Thanks @charleskawczynski for fixing the parameter tests.